ios/Video/RCTVideo.swift

```diff
@@ -1080,4 +1118,159 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     //         }
     //         */
     //    }
+
```

Comment:
Do you think this changes can be moved to a new file ?
Just try to keep code cleaner
Thanks

Reply:
Yes I'll try to move it to a new file! 😉

Reply:
I created a new file RCTIMAAdsManager under Features folder and put everything I could in it! 😉